docs/topics/create-first-yaml-rule.adoc

```diff
@@ -0,0 +1,196 @@
+.. To match on the `class-loading` element that is a child of `jboss-web`, use the XPath expression `jboss-web/web-loading` as an XML query. In this case, you need just one condition:
++
+[options="nowrap",subs="attributes+"]
+----
+when:
+  builtin.xml:
+    xpath: jboss-web/class-loading
+----
+<4> Helpful message explaining the migration issue. The message is generated in the report when the rule matches. For example:
++
+[options="nowrap",subs="attributes+"]
+----
+message: The class-loading element is no longer valid in the jboss-web.xml file.
+----
+<5> List of string labels for the rule
+<6> Number of expected story points to fix this issue
+<7> One or more hyperlinks pointing to documentation around the migration issues that you find
++
+[options="nowrap",subs="attributes+"]
+----
+links:
+- url: https://access.redhat.com/documentation/en-US/JBoss_Enterprise_Application_Platform/6.4/html-single/Migration_Guide/index.html#Create_or_Modify_Files_That_Control_Class_Loading_in_JBoss_Enterprise_Application_Platform_6
+  title: Create or Modify Files That Control Class Loading in JBoss EAP 6
+----
++
+The rule is now complete and looks similar to the following:
++
+[options="nowrap",subs="attributes+"]
+----
+- ruleID: jboss5-web-class-loading
+  description: Find class loading element in JBoss XML file.
+  when:
+    builtin.xml:
+      xpath: jboss-web/class-loading
+  message: The class-loading element is no longer valid in the jboss-web.xml file.
+  effort: 3
+  links:
+  - url: https://access.redhat.com/documentation/en-US/JBoss_Enterprise_Application_Platform/6.4/html-single/Migration_Guide/index.html#Create_or_Modify_Files_That_Control_Class_Loading_in_JBoss_Enterprise_Application_Platform_6
+    title: Create or Modify Files That Control Class Loading in JBoss EAP 6
+----
+
+[id="mta-installing-the-rule_{context}"]
+== Installing the rule
+
+.Procedure
+. Point the CLI  to the rule file you created :
++
+[options="nowrap",subs="attributes+"]
+----
+–rules /home/<USER>/rules.yaml
+----
+
+[id="mta-testing-the-rule_{context}"]
+== Testing the rule
+
+.Procedure
+To test the rule, point the input to the test data you created and pass the rule using the rules option in MTA CLI:
+
+[options="nowrap",subs="attributes+"]
+----
+mta-cli analyze --input /home/<USER>/data/ --output /home/<USER>/output/ --rules /home/<USER>/rules.yaml
+----
+
+[id="mta-reviewing-the-report_{context}"]
+== Reviewing the report
+
+Review the report to be sure that it provides the expected results.
+
+.Procedure
+
+. Once the analysis is complete, the command outputs the path to the HTML report:
++
+[options="nowrap",subs="attributes+"]
+----
+INFO[0066] Static report created. Access it at this URL:  URL="file:/home/<USER>/output/static-report/index.html"
+----
++
+Open `/home/<USER_NAME>/output/static-report/index.html` in a web browser.
+. Navigate to the *Issues* tab in the left menu.
```

Comment:
Does this open the MTA user interface? From "Open x" to "Navigate to the *Issues* tab" is a big jump.

Reply:
@RichardHoch 
I presume it does. It's the index page of a report.

Reply:
@sbeskin-redhat Consider adding a sentence saying that the MTA user interface opens (or whatever is correct). We did mention that they are opening the file in a browser, so up to you.